player/src/main/java/net/pubnative/player/VASTPlayer.java

```diff
@@ -634,6 +640,9 @@ public void onSkipClick() {
     public void onOpenClick() {
 
         VASTLog.v(TAG, "onOpenClick");
+        if (mBannerBitmap != null) {
+            mIsBannerSet = true;
+        }
```

Comment:
I don't get an idea of this fix. Why we not downloading if it null or did not set?